src/olympia/addons/migrations/0029_auto_20210118_1527.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.17 on 2021-01-18 15:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('addons', '0028_auto_20210114_1347'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='category',
+            name='addons',
+        ),
+        # migrations.RemoveConstraint(
+        #     model_name='addoncategory',
+        #     name='addon_id',
+        # ),
+        # migrations.RemoveIndex(
+        #     model_name='addoncategory',
+        #     name='category_addon_idx',
+        # ),
+        migrations.RemoveField(
+            model_name='addon',
+            name='categories',
+        ),
+        migrations.AlterField(
+            model_name='addoncategory',
+            name='category',
+            field=models.SmallIntegerField(),
+        ),
+        migrations.RenameField(
+            model_name='addoncategory',
+            old_name='category',
+            new_name='category_id',
```

Comment:
Can you avoid this ? This isn't backwards-compatible, that would cause loading/saving of addons to fail as soon as the migration is applied until the new code is deployed.

Reply:
the field in the database is already `category_id` because it's a foreign key, but the `AlterField` would rename it to `category` (when the field type changes), so this just changes it back.  There would be a moment during the migration where it database queries would fail though - I couldn't see a way to have AlterField not change the name and have it all be done in one step.